monitoring/alerting/consumer-rds.prometheusrulestests.template.yaml

```diff
@@ -0,0 +1,66 @@
+---
+$schema: /app-interface/prometheus-rule-test-1.yml
+
+rule_files:
+  - /TBD/PATH/TO/OUR/TEMPLATE/LOCATION
+
+evaluation_interval: 1m
+
+tests:
+# TransactionLogDiskSpaceUsageHigh
+- interval: 1m
+  input_series:
+  - series: aws_rds_transaction_logs_disk_usage_maximum{dbinstance_identifier="{{db_identifier}}"}
+    values: 2000000000x3 55000000000x5 # usage below threshold for 3 minutes, over aftera another 3
+
+  alert_rule_test:
+  # No alert in the first 3 mins
+  - eval_time: 3m
+    alertname: {{service}}_TransactionLogDiskSpaceUsageHigh
+    exp_alerts: []
+
+  # Alert after 7 min
+  - eval_time: 7m
+    alertname: {{service}}_TransactionLogDiskSpaceUsageHigh
+    exp_alerts:
+    - exp_labels:
+        severity: {{severity}}
+        service: {{service}}
+        dbinstance_identifier: {{db_identifier}}
+        env: {{env}}
+        app_team: {{app_team}}
+      exp_annotations:
+        dashboard: {{dashboard_link}}
+        link_url: {{console_link}}
+        message: "{{service}}/{{db_identifier}} RDS - Transaction log disk usage is high, which could indicate unchecked WAL growth or replication slot issues"
+        runbook: {{app_sop}}
+
+# ReplicationSlotLagOrUsageAnomaly
+- interval: 5m
+  input_series:
+  - series: aws_rds_transaction_logs_disk_usage_maximum{dbinstance_identifier="{{db_identifier}}"}
+    values: 0+67108860x12 # increases by 64GB every 5 mins --> 0.75GB per hour
+  - series: aws_rds_oldest_replication_slot_lag_maximum{dbinstance_identifier="{{db_identifier}}"}
+    values: 0+67108860x12 # increases by 64GB every 5 mins --> 0.75GB per hour
```

Comment:
Value looks right, just the comment itself

Reply:
Fixed in templates